sdk/servicebus/Azure.Messaging.ServiceBus/src/Diagnostics/ServiceBusEventSource.cs

```diff
@@ -1096,6 +1100,33 @@ public virtual void CreateManagementLinkException(string identifier, string exce
                 WriteEvent(ManagementLinkClosedEvent, identifier, linkException);
             }
         }
+
+        [Event(RequestAuthorizationStartEvent, Level = EventLevel.Verbose, Message = "{0}: Requesting authorization to {1}")]
+        public virtual void RequestAuthorizationStart(string identifier, string endpoint)
```

Comment:
I may be overlooking, but I'm not sure from usage how these differ from the `AmqpLinkAuthorizationRefresh` series (ids 39-41).    Should we consider merging them?